### ISSUE TITLE
Aspose Pdf 

### ISSUE THREAD

**Issue by Ashok772088** [aspose-free-consulting/projects#237]
While convert pdf to docx, empty tables showing as image ..How to resolve this?

**Comment by AlexNosk** [comment]
@Ashok772088 Could you please attach your input and output documents here for testing? We will check the issue and provide you more information.

**Comment by Ashok772088** [comment]
[Test.xlsx](https://github.com/aspose-free-consulting/projects/files/9220254/Test.xlsx)

I need same format with table editable in docx

**Comment by Ashok772088** [comment]
> @Ashok772088 Could you please attach your input and output documents here for testing? We will check the issue and provide you more information.

[Testing.xlsx](https://github.com/aspose-free-consulting/projects/files/9220326/Testing.xlsx)

I need editable table from xlsx to docx or pdf to docx

**Comment by AlexNosk** [comment]
@Ashok772088 You can consider using our sister company `GroupDocs` product `GroupDocs.Conversions`, which provides a functionality to convert [Excel to Word](https://docs.groupdocs.com/conversion/net/convert/spreadsheet/#convert-excel-workbook-to-docx).

**Comment by AlexNosk** [comment]
@Ashok772088 You can convert PDF to DOCX using Aspose.Words for .NET. See our documentation for more information:
<https://docs.aspose.com/words/net/convert-pdf-to-other-document-formats/>
Also, you can consider using our sister company's  GroupDocs.Conversions product, which provides a functionality to convert [Excel to Word](https://docs.groupdocs.com/conversion/net/convert/spreadsheet/#convert-excel-workbook-to-docx).

**Comment by Ashok772088** [comment]
@AlexNosk  I tried the above mentioned URL code already. 
1) Convert excel to pdf it is working as expected
2) Convert the PDF to Docx  we are not able to edit the empty table cells
3) Convert Excel to docx also same empty table cells issue
Is there any solution to fix this issue?

Note : Tried sister company Groupdocs also but the same issue occur there also

**Comment by AlexNosk** [comment]
@Ashok772088  Could you please attach your PDF document that causes the problem upon conversion from PDF to Word. I will check the issue and provide you more information. Regarding Excel to Docx conversion, I will ask my colleagues to take a look at this issue.

**Comment by atirtahirgroupdocs** [comment]
@Ashok772088 


> > @Ashok772088 Could you please attach your input and output documents here for testing? We will check the issue and provide you more information.
> 
> [Testing.xlsx](https://github.com/aspose-free-consulting/projects/files/9220326/Testing.xlsx)
> 
> I need editable table from xlsx to docx or pdf to docx

This issue (Excel to Word conversion, keep the table editable) is reproduced at our end using GroupDocs.Conversion API. Therefore, we have logged it in our internal issue tracking system. You'll be notified in case of any update.

**Comment by amjad-sahi** [comment]
@AlexNosk,

I tested your scenario/case using our latest version/fix: Aspose.Cells for .NET v22.7 ([Download](https://releases.aspose.com/cells/net/) | [NuGet](https://www.nuget.org/packages/Aspose.Cells/)) to convert your template Excel file (Testing.xlsx") to DOCX and it works fine as I tested. The empty cells are editable in the output DOCX file in MS Word. I used the following sample code using your template file with Aspose.Cells for .NET v22.7:
e.g.
**Sample code:**
```cs
            Workbook workbook = new Workbook("e:\\test2\\Testing.xlsx");
            workbook.Save("e:\\test2\\out1.docx", Aspose.Cells.SaveFormat.Docx);
```
Please find attached the output DOCX file for your reference. 

[out1.docx](https://github.com/aspose-free-consulting/projects/files/9251389/out1.docx)

Let us know if you still find any issue.

**Comment by Ashok772088** [comment]
@amjad-sahi  :  Thanks for your reply.  I tried you approach, cell can be editable , Suppose if remove the gridlines and border for some cells in excel, while convert excel to docx i am getting the gridlines and border for all the cells. How to fix this issue?

**Comment by amjad-sahi** [comment]
@Ashok772088,

Yes, this is limitation or bug in Excel to DOCX conversion and we logged a ticket for it in the forums. You may post your issue/query or follow up in the dedicated [forums](https://forum.aspose.com/c/cells/9).

**Comment by Ashok772088** [comment]
is any other way to achieve this fix?

**Comment by amjad-sahi** [comment]
@Ashok772088 ,

Please check my previous reply and do the needful to post your issue in the dedicated forums. If you are talking about removing gridlines or borders in the output DOCX, as I told you this is a limitation in Excel to DOCX conversion and will not be fixed in near future.

**Comment by amjad-sahi** [comment]
@Ashok772088,

We improved Excel to DOCX rendering in latest version. Please try our latest version/fix: Aspose.Cells for .NET v22.9 ([Download](https://releases.aspose.com/cells/net/) | [NuGet](https://www.nuget.org/packages/Aspose.Cells/)). Your issue should be fixed in it.

Let us know if you still find any issue.

**Comment by atirtahirgroupdocs** [comment]
> @Ashok772088
> 
> > > @Ashok772088 Could you please attach your input and output documents here for testing? We will check the issue and provide you more information.
> > 
> > 
> > [Testing.xlsx](https://github.com/aspose-free-consulting/projects/files/9220326/Testing.xlsx)
> > I need editable table from xlsx to docx or pdf to docx
> 
> This issue (Excel to Word conversion, keep the table editable) is reproduced at our end using GroupDocs.Conversion API. Therefore, we have logged it in our internal issue tracking system with ticket ID CONVERSIONNET-5395. You'll be notified in case of any update.

Please try GroupDocs.Conversion for .NET 23.4. Table in the converted document is editable. 
Thanks
